scripts/zones/Ordelles_Caves/npcs/Ruillont.lua

```diff
@@ -23,7 +23,7 @@ function onTrigger(player, npc)
     if player:getCurrentMission(SANDORIA) == THE_RESCUE_DRILL then
         local missionStatus = player:getVar("MissionStatus")
 
-        if missionStatus == 7 then
+        if missionStatus <= 7 then
```

Comment:
This will cause skipping steps 1 through 7 if the player talks to this NPC after the mission has been started, as the [eventFinish for cs 1](https://github.com/DarkstarProject/darkstar/blob/945ce35e64e518d3a2fbb75787d658ef0e005c5f/scripts/zones/Ordelles_Caves/npcs/Ruillont.lua#L44) advances the mission status to 8.

Reply:
If talking to this NPC is strictly optional and shouldn't influence the quest, the check here is fine, but you'll have to remove the mission status update in event 1, and either:
1) Find whatever NPC comes before this one and have it skip over 7 straight to 8; or
2) Find all NPCs that come after this one and decrement the required mission statuses to trigger them (and what they set the mission status too in the event finish) _after_ the player reaches mission status 7 by 1.